dev/io.openliberty.threading.virtual/src/com/ibm/wsspi/threading/VTOverrideService.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.wsspi.threading;
+
+/**
+ * This spi allows disabling the usage of VirtualThreads within the control of the Managed Executor framework
```

Comment:
Although ManagedExecutorService, ManagedScheduledExecutorService, and ManagedThreadFactory are currently the only API that allow users to request Liberty to create virtual threads, the intent is for there to be other ways of requesting in the future. Jakarta EE 12 even has a special annotation that is being proposed for this.
```suggestion
 * This SPI allows disabling the creation of virtual threads within the control of Liberty.
 * When creation of virtual threads is disabled, Liberty will create platform threads instead.
```